app/controllers/idt/api/v1/appeals_controller.rb

```diff
@@ -6,6 +6,10 @@ class Idt::Api::V1::AppealsController < Idt::Api::V1::BaseController
 
   skip_before_action :verify_authenticity_token, only: [:outcode]
 
+  rescue_from BGS::AccountLocked do |_e|
+    render(json: { message: "Your account is locked.  Please contact your Security Officer" }, status: :forbidden)
```

Comment:
Should we be directing the user to ESD, mentioned [here](https://dsva.slack.com/archives/CD5DAQNCU/p1593618517119500)?

Reply:
```suggestion
    render(json: { message: "Your account is locked. Please contact the VA Enterprise Service Desk to resolve this issue." }, status: :forbidden)
```

What about this? You'd have to update the test as well.

Reply:
Yoom, I like the idea of mentioning the ESD instead of their Security Officer

Reply:
any reason we shouldn't put this in [the base controller](https://github.com/department-of-veterans-affairs/caseflow/blob/master/app/controllers/idt/api/v1/base_controller.rb) instead of the appeals controller?